pallets/identity/src/lib.rs

```diff
@@ -115,10 +115,10 @@ decl_storage! {
         Owner get(fn owner) config(): T::AccountId;
 
         /// DID -> identity info
-        pub DidRecords get(fn did_records) config(): map IdentityId => DidRecord;
+        pub DidRecords get(fn did_records) config(): map hasher(blake2_256) IdentityId => DidRecord;
```

Comment:
All of these will have to be changed in a new PR :sweat_smile:

Reply:
Yes you are right